### PR TITLE
Add polarion test_id flags

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
They are needed to filter correctly the test from polarion test plan and also dump
polarion xml.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by geetikakay** [comment]
looks like one test case is missing.

CNV-2178 [Network][KMP] MAC range generated after kubemacpool installation (Tire 1) could be because tier spelling mistake

**Comment by qinqon** [comment]
@geetikakay that test feels a CNAO tier1 tests more than kubemacpool, what do you think @phoracek ?

File: tests/virtual_machines_test.go [hunk 1]
```diff
@@ -141,7 +142,7 @@ var _ = Describe("Virtual Machines", func() {
 		Context("When the client tries to assign the same MAC address for two different vm. Within Range and out of range", func() {
 			//2166
```
[the remark this answers is not in the document]
Comment (qinqon) [reply tests/virtual_machines_test.go]: done

**Pull request merged**